### DISCUSSION TITLE
question: Diff pane: is there a command to jump between hunks or scroll a set number of lines?

### DISCUSSION THREAD

**Discussion by MagicDuck** [extrawurst/gitui#939] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Was looking for a command to jump between hunks, I see page_up & page_down, but nothing more granular (except <up> and <down> arrows)

**Describe the solution you'd like**
some command to move between hunks or scroll up/down like 10 lines at a time

**Describe alternatives you've considered**
not sure there's any

**Additional context**

**Comment by extrawurst** [comment, marked as answer]
No we do not have such a command. We also have no way to define such combo commands necessary to do that. So far I never missed such a feature since we have up/down, paging and home/end. Might be more a habit for vim users and the fact that this is a common navigation method there.

**Reply by extrawurst** [reply]
as of #1523 this will be soon released in 0.24

**Comment by fceruti** [comment]
Easily one of my favorite TUI software, thanks for building this!

To the topic in question:
Maybe it's just me, but I would really appreciate Next & Previous Hunk commands.
